include/shardwrapper.hpp

```diff
@@ -46,6 +46,8 @@ template <class T> struct ShardWrapper {
   static __cdecl Shard *create() {
     Shard *result = reinterpret_cast<Shard *>(new (std::align_val_t{16}) ShardWrapper<T>());
 
+    result->refCount = 0;
```

Comment:
Well the allocation above will also zero memory so no need.

Reply:
Are you sure? shardwrapper doesn't seem to initialize the header struct?

Reply:
Removed it does